### PR TITLE
Add tests for ec2_vpc_route_table

### PR DESCRIPTION
**Pull request by willthames**
##### SUMMARY
Initial tests for ec2_vpc_route_table ready for boto3 porting

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
ec2_vpc_route_table

##### ANSIBLE VERSION
```
ansible 2.5.0 (devel 6693cc3f5a) last updated 2017/10/23 18:16:53 (GMT +1000)
  config file = None
  configured module search path = [u'/Users/will/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/will/src/ansible/lib/ansible
  executable location = /Users/will/src/ansible/bin/ansible
  python version = 2.7.13 (default, Dec 18 2016, 07:03:39) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```

### CODE REVIEW

**Comment by willthames** [comment]
@mattclay extra permissions are listed in the attached ec2-policy.json

File: test/integration/targets/ec2_vpc_route_table/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,471 @@
+- block:
+
+  - name: create VPC
+    ec2_vpc_net:
+      cidr_block: 10.228.228.0/22
+      name: "{{ resource_prefix }}_vpc"
+      state: present
+      region: "{{ aws_region }}"
+      aws_access_key: "{{ aws_access_key|default(omit) }}"
```
Comment (s-hertel) [inline test/integration/targets/ec2_vpc_route_table/tasks/main.yml]: Can you set these as environment vars on the block, please? To test insufficient credentials, make another block.
Example:
https://github.com/ansible/ansible/pull/31892/files#diff-b7e5b439f984647e05df1d4531f0b626R69

**Review by s-hertel — CHANGES_REQUESTED** [review]

File: test/integration/targets/ec2_vpc_route_table/tasks/main.yml (hunk 1 above)
Comment (willthames) [reply test/integration/targets/ec2_vpc_route_table/tasks/main.yml]: Is there a reason this is preferred over task parameters? Is the preference documented anywhere?

**Review by s-hertel — APPROVED** [review]

**Comment by s-hertel** [comment]
bot_status

File: test/integration/targets/ec2_vpc_route_table/tasks/main.yml (hunk 1 above)
Comment (willthames) [reply test/integration/targets/ec2_vpc_route_table/tasks/main.yml]: I'll go with YAML as per IRC

**Pull request merged**